tests/CMakeLists.txt

```diff
@@ -202,8 +202,9 @@ if(WITH_PYTHON)
   add_fdb_test(TEST_FILES fast/RandomUnitTests.toml)
   add_fdb_test(TEST_FILES fast/ReadHotDetectionCorrectness.toml IGNORE) # TODO re-enable once read hot detection is enabled.
   add_fdb_test(TEST_FILES fast/ReportConflictingKeys.toml)
-  add_fdb_test(TEST_FILES fast/RESTKmsConnectorUnit.toml IGNORE)
-  add_fdb_test(TEST_FILES fast/RESTUtilsUnit.toml IGNORE)
+  add_fdb_test(TEST_FILES fast/RESTKmsConnectorUnit.toml)
+  add_fdb_test(TEST_FILES fast/RESTSimKmsVaultUnit.toml)
+  add_fdb_test(TEST_FILES fast/RESTUtilsUnit.toml)
```

Comment:
These tests were ignored because they were very fast-running which caused joshua to run them too many times. These should explicitly not be in their own test files and should be ran as part of the other unit test mechanisms (RandomUnitTest/SpecificUnitTest) unless there is some reason they can't be run as part of those.

Reply:
Ahh I might have forgotten the details. Will leave them IGNOR-ed and them running as part of RandomUnitTest works fine